Applications/DataExplorer/DataView/NetCdfConfigureDialog.cpp

```diff
@@ -43,7 +41,7 @@ NetCdfConfigureDialog::~NetCdfConfigureDialog()
 void NetCdfConfigureDialog::accept()
 {
 	QMessageBox valueErrorBox;
-	if (_currentVar->num_dims() < 3){
+	if (_currentVar->num_dims() < 2){
```

Comment:
Thank you very much! I think it would be extremly helpful if you could transform the explanation to the code somehow.